src/Arcus.Messaging.Abstractions/IMessageCorrelationInfoAccessor.cs

```diff
@@ -1,11 +1,13 @@
-﻿using Arcus.Observability.Correlation;
+﻿using System;
+using Arcus.Observability.Correlation;
 
 namespace Arcus.Messaging.Abstractions
 {
     /// <summary>
     /// Represents a marker interface for an <see cref="ICorrelationInfoAccessor{TCorrelationInfo}"/> implementation using the messaging <see cref="MessageCorrelationInfo"/>.
     /// </summary>
     /// <seealso cref="ICorrelationInfoAccessor{TCorrelationInfo}"/>
+    [Obsolete("Will be moved in v3.0 outside the 'Abstractions' library in a separate Serilog-specific library, see the v3.0 migration guide for more information")]
```

Comment:
Is it possible to do this now already ?
As, imho, it makes no sense to deprecate a class now, without having the alternative ready ?
Or, we must make sure that the obsolete class is still part of the library when the alternative is thre

Reply:
1. If we leave the class non-deprecated, we have to wait for v4 to remove it;
2. The alternative will be available on v3, as it will include breaking changes to include an alternative.
As far as I see, there is no way of doing both removing all references (Serilog) and yet still provide support for all references.

Reply:
Ok, fine for me.